content/docs/setup/kubernetes/install/kubernetes/index.md

```diff
@@ -126,17 +133,19 @@ $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
 
     {{< text bash >}}
     $ kubectl get pods -n istio-system
-    NAME                                      READY     STATUS      RESTARTS   AGE
-    istio-citadel-5c4f467b9c-m8lhb            1/1       Running     0          8d
-    istio-cleanup-secrets-1.1.0-rc.0-msbk7    0/1       Completed   0          8d
-    istio-egressgateway-fbfb4865d-rv2f4       1/1       Running     0          8d
-    istio-galley-7799878d-hnphl               1/1       Running     0          8d
-    istio-ingressgateway-7cf9598b9c-s797z     1/1       Running     0          8d
-    istio-pilot-698687d96d-76j5m              2/2       Running     0          8d
-    istio-policy-55758d8898-sd7b8             2/2       Running     3          8d
-    istio-sidecar-injector-5948ffdfc8-wz69v   1/1       Running     0          8d
-    istio-telemetry-67d8545b68-wgkmg          2/2       Running     3          8d
-    prometheus-c8d8657bf-gwsc7                1/1       Running     0          8d
+    NAME                                      READY   STATUS    RESTARTS   AGE
+    grafana-85d8646cb-xfbdn                   1/1     Running   0          6m
+    istio-citadel-5c85447bdd-pn8ns            1/1     Running   0          6m
+    istio-egressgateway-65f8c57947-2gb98      1/1     Running   0          6m
+    istio-galley-7fc6658847-kjnfq             1/1     Running   0          6m
+    istio-ingressgateway-6bbbccf8f4-wwknm     1/1     Running   0          6m
+    istio-pilot-5d76cd68bb-vj59r              2/2     Running   0          6m
+    istio-policy-6c86dc575d-nnz8w             2/2     Running   0          6m
+    istio-sidecar-injector-745794f57f-wjzdf   1/1     Running   0          6m
+    istio-telemetry-5df96b9557-7jb4d          2/2     Running   0          6m
+    istio-tracing-7dbc876499-g7lvc            1/1     Running   0          6m
+    kiali-5d579bf6d8-trp2k                    1/1     Running   0          6m
+    prometheus-6cdfbb4746-7494d               1/1     Running   0          6m
```

Comment:
Do you see any post-install pods with `Completed` status?

Reply:
Strange - I don't see any jobs in my `get pods` output. I can try again on a new install.

Reply:
Are you using the latest daily build?

Reply:
Fixed using the latest build. Thanks